.github/workflows/build-webapi.yaml

```diff
@@ -11,8 +11,8 @@ on:
 
 env:
   DOCKERFILE_PATH: "CarbonAware.WebApi/Dockerfile"
-  HEALTH_ENDPOINT: "0.0.0.0:8080/emissions/bylocation?location=eastus"
-  #Health Check Endpoint to be added as another story and replaced here
+  HEALTH_ENDPOINT: "0.0.0.0:8080/health"
+  DATA_ENDPOINT: "0.0.0.0:8080/emissions/bylocation?location=eastus"
```

Comment:
Can you explain what you're hoping to test with this?  I was expecting the health check to replace this endpoint